Makefile

```diff
@@ -9,7 +9,6 @@ PACKAGE = asyncjs
 REQUIRE_NAME = async
 UGLIFY = uglifyjs
 XYZ = support/xyz.sh --repo git@github.com:caolan/async.git
-COPY_ES = sed -E "s/(import.+)lodash/\1lodash-es/g"
```

Comment:
I don't think you can remove this, I think it actually becomes just a simple `cp`

Reply:
I'm sure of what it does exactly, what would the `cp` command be exactly?

Reply:
Down below we need to copy files from `lib/` to `build-es/`.  `$(COPY_ES)` literally just becomes `cp`.

Reply:
Err, `cp` minus the pipe.  Guess `cat` would work in that case, but it's probably best to re-work the (removed) commands below slightly.

Reply:
I'm not familiar with how Make works, so it's perhaps best to push a commit if you have time :)